src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/AdapterSwitchItem.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+
+type Props = {
+    adapter: string,
+    active: boolean,
+    onClick: (adapter: string) => void,
+};
+
+export default class AdapterSwitchItem extends React.PureComponent<Props> {
+    handleClick = () => {
+        const {
+            adapter,
+            onClick,
+        } = this.props;
+
+        onClick(adapter);
+    };
+
+    render() {
+        const {
+            active,
+            adapter,
+        } = this.props;
+
+        return (
+            <li key={adapter} onClick={this.handleClick}>
+                {active ? (
+                    <i>{adapter}</i>
```

Comment:
Why an `i` tag? I would make that at least an `a` tag, so that it is something clickable. Actually I think I would make a simple select here. That's not a lot of work, and would already look a lot nicer. I know that it will be rebuilt anyway, but the `develop` branch is already tested by a few developers out there, therefore I would try to not forget about this stuff completely 🙈